src/Test/Utilities/Portable/Traits/CompilerFeature.cs

```diff
@@ -35,5 +35,6 @@ public enum CompilerFeature
         DefaultInterfaceImplementation,
         LambdaDiscardParameters,
         StatementAttributes,
+        InitOnly,
```

Comment:
Nit: Consider calling this `InitOnlySetters`. This will work better in a future release if we add the more general init only member support. That second feature may be better called `InitOnly`